src/Styleguide/Pages/Artwork/Sidebar.tsx

```diff
@@ -50,6 +60,8 @@ export class Sidebar extends Component<ArtworkSidebarProps> {
         <ArtworkMetadata artwork={artwork} />
         <Separator />
         <Commercial artwork={artwork} />
+        <PartnerInfo partner={artwork.partner} artwork={artwork} />
```

Comment:
Debated for a while if this component should take just partner and always display partner info. In this case check for `artwork.collecting_institution` would have to happen here and somehow that did not feel correct... Open for suggestions:)